aws/service.ts

```diff
@@ -131,42 +131,6 @@ function createLoadBalancer(
     };
 }
 
-// getImageName generates an image name from a container definition.  It uses a combination of the container's name and
-// container specification to normalize the names of resulting repositories.  Notably, this leads to better caching in
-// the event that multiple container specifications exist that build the same location on disk.
-function getImageName(container: cloud.Container): string {
```

Comment:
overall idea.  we used to switch on the type of hte container twice.  once to get an image name, once to actually build things.  now this is all into once switch.  That makes things simple since it means that the only time we need a repository is when we have a .build.  ANd if we have a .build, then we know we can build a string imageName instead of dealing with aan Input<string>